packages/report-server/src/app/createApp.ts

```diff
@@ -38,6 +39,11 @@ export function createApp(database: TupaiaDatabase, models: ModelRegistry) {
     next();
```

Comment:
I think we may now be able to remove setting the authenticator on the request object above?

Reply:
Good point, that cleans things up :-)